testHelpers/hasAllKeys.js

```diff
@@ -0,0 +1,14 @@
+const hasAllKeys = (obj, array) => {
+  let objKeys = Object.keys(obj);
+
+  for ( let i = 0; i < objKeys.length; i++) {
+    if (objKeys[i] !== array[i]) {
+      return false
+    }
```

Comment:
Would it be possible if Object.keys(obj) has the same keys as array but in different order? If so, would you want to return false?

Reply:
It might be?  I changed it to .includes instead.  Thank you!